modules/candidate_parameters/test/TestPlan.md

```diff
@@ -81,3 +81,37 @@
 38. Check that entering a date of death before the date of birth gives a data entry error.
 39. Check that entering a date later than today gives a data entry error.
 40. Check that the date stored matches the _DOD Format_ specified in the configuration module.
+
+## Widgets
+41. Access a candidate in the `candidate_profile` module
+
+### Candidate Info Widget terms
+42. Within the "Candidate Info" card, after the information which
+    corresponds to the Candidate info returned by the LORIS API:
+43. Verify that there is a "Participant Status" term with the candidate's
+    status which matches the Parameter Status tab in the `candidate_parameters`
+    module or "N/A" if there is no status for this candidate.
+44. Verify that there is a "Caveat" term on the card with the
+    description of the reason for a caveat which matches the
+    Caveat Emptor flag of the "Candidate Info" tab of the
+    `candidate_parameters` module, and that the term does *not*
+    appear if the candidate does not have a caveat.
+45. Verify that any terms from the "Candidate Parameters"
+    `parameter_type_category` in step 5 of the test plan are displayed
+    in the "Candidate Info" card.
+
+### Consent Widget
+46. With Consent configured in LORIS, verify that there is a
+   "Consent" card in the Candidate Profile widget.
+47. Verify that the card does *not* appear is Consent is
+    not configured in LORIS.
```

Comment:
```suggestion
    not enabled in LORIS.
```

Reply:
See other comment.. there is no enable/disable consent in LORIS as far as I know.

Reply:
Ok.  I don't think it's clearer now but at least it's consistent 👍